src/android/hooks/configure-android.js

```diff
@@ -7,9 +7,9 @@ module.exports = function (ctx) {
     if (ctx.opts.platforms.indexOf('android') < 0) {
         return;
     }
-    var fs = ctx.requireCordovaModule('fs'),
-        path = ctx.requireCordovaModule('path'),
-        deferral = ctx.requireCordovaModule('q').defer();
+    var fs = require('fs'),
+        path = require('path'),
+        deferral = require('q').defer();
```

Comment:
Same comment as https://github.com/servicetitan/cordova-plugin-ms-adal/pull/1#discussion_r354956315

Reply:
I didn't notice this initially but this `deferral` is actually unused so I just removed it